src/index.ts

```diff
@@ -87,6 +87,7 @@ const RESERVED_WORDS = [
 	'try', 'typealias', 'unowned', 'var', 'weak', 'where', 'while', 'willSet',
 	'LocalDate', 'LocalTime', 'OffsetDateTime', 'Decimal', 'String', 'Void', 'File', 'FormData',
 	'unknown', // for our enum cases
+	'RetryConfiguration',
```

Comment:
This does make us think... what other words are we missing? `Configuration`?

Reply:
I've added all the types from our Oauth too